src/clients/mockApi.ts

```diff
@@ -116,6 +120,26 @@ export async function postIssue(issuePostBody: IssuePostBody): Promise<Issue> {
   return setInLocalStorage(createIssue(issuePostBody))
 }
 
+export async function getSiteData(site: string): Promise<SiteData | null> {
+  try {
+    const issues = await searchIssues(site)
+    return {
+      site: defaultSite,
+      maintainer: {
+        avatarUrl: '/devdiva.png',
+        username: 'devdiva22',
+      },
+      issues: {
```

Comment:
I'm realizing that choosing the uppercase statuses was a bad choice. I'm pushing a commit to rectify this as it's already cluttered the code in a number of areas.

The reason I bring that up is that with that in place `groupBy` from lodash will achieve this same effect nicely.